### PR TITLE
Add UpdateUserStatus and DeleteUserSSOToken functionality

### PR DESCRIPTION
**Pull request by astockwell**
Added support for the following User APIs:

- UpdateUserStatus() - https://marketplace.zoom.us/docs/api-reference/zoom-api/users/userstatus
- DeleteUserSSOToken() - https://marketplace.zoom.us/docs/api-reference/zoom-api/users/userssotokendelete

Tests included and validated in vivo.

### CODE REVIEW

**Comment by astockwell** [comment]
Possibly a silly question - I'm not sure how to get the Zoom-native tests to pass Travis CI, is that necessary to be merged?

**Comment by rafecolton** [comment]
@astockwell good question, they should pass 😅 It looks like I need to update the credentials, let me check into that and get back to you. Thanks for the PR!

File: user_test.go [hunk 1]
```diff
@@ -1,5 +1,3 @@
-// +build integration
```
Comment (rafecolton) [inline user_test.go:1]: Why was this removed?

File: user_update_status.go [hunk 2]
```diff
@@ -0,0 +1,32 @@
+package zoom
```
Comment (rafecolton) [inline user_update_status.go:1]: This looks like an integration test, it needs a build tag comment

File: user_test.go (hunk 1 above)
Comment (astockwell) [reply user_test.go:1]: Apologies, that was an oversight!

**Comment by rafecolton** [comment]
Closing due to no update in over a year. Feel free to reopen if you are still actively working on this.
